test/e2e/app/math.go

```diff
@@ -0,0 +1,10 @@
+package app
+
+func Sum(batches []int) uint64 {
+	var resp int
+	for _, b := range batches {
+		resp += b
+	}
+
+	return uint64(resp)
+}
```

Comment:
Sum doesn't need to be exported. Also it doesn't need to know that it sums batches (it can be any slice of ints).
```suggestion
func sum(v []int) uint64 {
	var res int
	for _, n := range v {
		res += n
	}

	return uint64(res)
}
```